src/search/FindBar.js

```diff
@@ -333,6 +346,15 @@ define(function (require, exports, module) {
                         // if Shift is held down).
                         self.trigger("doFind", e.shiftKey);
                     }
+                    currentIndex = 0;
+                } else if (e.keyCode === KeyEvent.DOM_VK_DOWN) {
+                    currentIndex = (currentIndex - 1 + Math.min(maxCount, searchHistory.length)) % Math.min(maxCount, searchHistory.length);
+                    $("#find-what").val(searchHistory[currentIndex]);
+                    self.trigger("queryChange");
+                } else if (e.keyCode === KeyEvent.DOM_VK_UP) {
+                    currentIndex = (currentIndex + 1 + Math.min(maxCount, searchHistory.length)) % Math.min(maxCount, searchHistory.length);
```

Comment:
can you wrap `if (e.keyCode === KeyEvent.DOM_VK_DOWN)` and `if (e.keyCode === KeyEvent.DOM_VK_UP) {` codes into a function taking an argument up or down? code seems to repeat itself